api/lib/application/security-pre-handlers.js

```diff
@@ -242,6 +243,18 @@ async function checkUserIsMemberOfAnOrganization(request, h) {
   return _replyForbiddenError(h);
 }
 
+async function checkUserIsAdminInOrganizationOrOwnerOfTheCampaign(request, h) {
+  const userId = request.auth.credentials.userId;
+  const campaignId = parseInt(request.params.id);
```

Comment:
Je me demande si la validation joi a pas lieu avant le préhandler ?
Car sinon, ça va sortir une 400 et il n'y a pas besoin de faire un parseInt (y'a des tests api/tests/unit/application/campaign/index_test.js sur le should return 400 with an invalid campaign id avec le PATCH )

Reply:
en effet, j'essaie de l'enlever voir si ça passe :)

Reply:
Done ✅